.aws/src/config/index.ts

```diff
@@ -9,7 +9,11 @@ const log_retention_days = 30;
 const agentCpu = 1024;
 const agentMemory = 2048;
 const agentTaskCount = 1;
-const agentImage = 'prefecthq/prefect:2-python3.10';
+const OIDCOrgId = process.env['OIDC_ORG_ID'] || '';
+const OIDCProjectId = process.env['OIDC_PROJECT_ID'] || '';
+const gitSha = process.env['CIRCLE_SHA1'] || 'dev';
+const imageTag = gitSha.slice(0, 7);
+const vpcName = isDev ? 'SharedVpc' : 'Pocket VPC';
```

Comment:
The OICD configs are to support Open ID Connect access to AWS rather than using a self hosted runner or access keys.

gitSha and imageTag are for the referencing the proper agent image in ECR

vpcName is needed for creating security groups and rules